src/rf/apps/workers/thumbnail.py

```diff
@@ -138,9 +145,6 @@ def make_thumbs_for_layer(layer_id):
     except ImageCouldNotOpenError:
         return False
 
-    layer.status = enums.STATUS_THUMBNAILED
```

Comment:
This was moved to processing. I wanted to remove non-thumbnail related tasks (such as updating the status) but without more major overhaul it is necessary to allow this to update each individual image. I compromised and moved the layer status out to processing.py.